packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -302,27 +302,7 @@ class FlutterPlugin implements Plugin<Project> {
                 }
             }
 
-            GenerateDependencies dependenciesTask = project.tasks.create(name: "flutterDependencies${variant.name.capitalize()}", type: GenerateDependencies) {
-                flutterRoot this.flutterRoot
-                flutterExecutable this.flutterExecutable
-                buildMode flutterBuildMode
-                localEngine this.localEngine
-                localEngineSrcPath this.localEngineSrcPath
-                targetPath target
-                previewDart2 previewDart2Value
-                fileSystemRoots fileSystemRootsValue
-                fileSystemScheme fileSystemSchemeValue
-                trackWidgetCreation trackWidgetCreationValue
-                preferSharedLibrary preferSharedLibraryValue
-                targetPlatform targetPlatformValue
-                sourceDir project.file(project.flutter.source)
-                intermediateDir project.file("${project.buildDir}/${AndroidProject.FD_INTERMEDIATES}/flutter/${variant.name}")
-                extraFrontEndOptions extraFrontEndOptionsValue
-                extraGenSnapshotOptions extraGenSnapshotOptionsValue
-            }
-
             FlutterTask flutterTask = project.tasks.create(name: "flutterBuild${variant.name.capitalize()}", type: FlutterTask) {
-                dependsOn dependenciesTask
```

Comment:
Do you know what was the purpose of flutterDependencies task? It sounds like it was to generate initial dependency file for this flutterBuild task, so that flutterBuild task can be skipped if sources haven't changed.

Reply:
That was the purpose, yes, but since then some other changes were made to Flutter Tools, so it now also checks if the source files have changed before building a new snapshot. So it's no longer necessary to do a double build in Gradle anymore.